src/app/editor/viewer/lib/sketch.service.ts

```diff
@@ -45,42 +45,42 @@ export class SketchService {
   }
 
   async process(file: File) {
-    try {
-      this._data = await this.sketch2Json(file);
-      if (this.sketchStyleParser.visit(this._data) === SupportScore.LEGACY) {
-        this.store.dispatch(
-          new InformUser(
-            'The design was created using a legacy version so the result may not be accurate.'
-          )
-        );
-      }
-    } catch (e) {
+    this._data = await this.sketch2Json(file);
+    if (this.sketchStyleParser.visit(this._data) === SupportScore.LEGACY) {
       this.store.dispatch(
         new InformUser(
-          'The design was created using an unsupported version.'
+          'The design was created using a legacy version so the result may not be accurate.'
```

Comment:
The design was created using a legacy version of SketchApp, so the result may not be accurate.